usr/bin/hwctl

```diff
@@ -321,16 +321,30 @@ function display {
 }
 
 function system-battery {
+	if [ -e /sys/class/power_supply/qcom-battmgr-bat/status ]; then
+		BAT_PATH='/sys/class/power_supply/qcom-battmgr-bat'
+	elif [ -e /sys/class/power_supply/BAT1/status ]; then
+		BAT_PATH='/sys/class/power_supply/BAT1'
+	elif [ -e /sys/class/power_supply/BAT0/status ]; then
```

Comment:
Nice, BAT0 also happens to work for the ASUS ROG Ally (not just the AYANEO 2)!

The Lenovo Legion Go uses a different location: `BATT`.

```
$ cat /sys/class/power_supply/BATT/capacity
99
```

Could we do a wildcard here to check for `/sys/class/power_supply/BAT*/capacity`? It might not work if a system has more than one battery (some enterprise laptops will) but that could be fine for now.